File: Python/sorting_algorithms/SelectionSort.py
def SelectionSort(list):
    for i in range(0, len(list)):
        sel = list[i]
        selId = i
        for j in range(i, len(list)):
            if list[j] < sel:
                sel = list[j]
                selId = j
        list[i], list[selId] = list[selId], list[i]

    return list

File: Rust/sorting_algorithms/src/selection_sort.rs
pub fn sort(list: &mut [i32]) {
    for i in 0..list.len() {
        let mut sel: i32 = list[i];
        let mut sel_id = i;
        for j in i..list.len() {
            if list[j] < sel {
                sel = list[j];
                sel_id = j
            }
        }
        list.swap(i, sel_id);
    }
}
